tests/test_benchmark_models.py

```
@@ -0,0 +1,236 @@
+
+    Charging every action for the full chunk would overstate the budget by the number of
+    actions, which is exactly what makes the column worth reporting.
+    """
+
+    class _Net:
+        @staticmethod
+        def compute_epoch_steps(n_samples):
+            return [n_samples]  # one step per row, so steps == rows seen
+
+    class _Mab:
+        actions = {"a0": _Net()}
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)